src/m365/spo/commands/tenant/tenant-recyclebinitem-restore.ts

```diff
@@ -113,7 +95,6 @@ class SpoTenantRecycleBinItemRestoreCommand extends SpoCommand {
       // This has to be removed in the next major release.
       await logger.log({
```

Comment:
@nanddeepn as mentioned in the issue, we should remove the entire command output. Could you fix this please?